lib/manager/components/ProjectSettingsForm.js

```diff
@@ -274,25 +274,27 @@ export default class ProjectSettingsForm extends Component<Props, State> {
     return (
       <div>
         <ConfirmModal ref='confirm' />
-        <Panel header={<h4>{this.messages('title')}</h4>}>
+        <Panel>
+          <Panel.Heading><Panel.Title componentClass='h3'>{this.messages('title')}</Panel.Title></Panel.Heading>
           <ListGroup fill>
             <ListGroupItem>
               <FormGroup
                 data-test-id='project-name-input-container'
                 validationState={validationState(validation.name)}>
                 <ControlLabel>{this.messages('fields.name')}</ControlLabel>
                 <FormControl
-                  value={model.name || ''}
                   name={'name'}
                   onChange={this._onChangeTextInput}
+                  value={model.name || ''}
                 />
                 <FormControl.Feedback />
                 <HelpBlock>Required.</HelpBlock>
               </FormGroup>
             </ListGroupItem>
           </ListGroup>
         </Panel>
-        <Panel header={<h4>{this.messages('fields.updates.title')}</h4>}>
+        <Panel>
+          <Panel.Heading><Panel.Title componentClass='h3'>{this.messages('fields.updates.title')}</Panel.Title></Panel.Heading>
```

Comment:
Same thing as before, I assume it probably doesn't make much of a difference, but this was an `h4` before

Reply:
changed!